test/Datadog.Trace.ClrProfiler.IntegrationTests/PackageVersions.g.cs

```diff
@@ -278,5 +278,219 @@ public class PackageVersions
             };
 #endif
 
+        public static IEnumerable<object[]> AmazonSqs =>
+
+            new List<object[]>
+            {
+#if DEFAULT_SAMPLES
+                new object[] { string.Empty },
+#else
+                new object[] { "3.1.0" },
```

Comment:
This list is another reason I want to limit how many packages we test. This is a little ridiculous.

Reply:
see what you mean